packages/body/tests/site/index.html

```diff
@@ -0,0 +1,125 @@
+		</script>
+	<![endif]-->
+</head>
+<body>
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
```

Comment:
do we add it to a `/test` or `/examples` folder in that case. Still unsure about how all the bits piece together so could be also talking 💩